src/server/views/summary.html

```diff
@@ -18,6 +19,13 @@
         }) }}
       {% endif %}
 
+      {% if notificationEmail is undefined %}
```

Comment:
Probably good to check for empty values too

```suggestion
      {% if not notificationEmail | length %}
```

Reply:
agree, done